### ISSUE TITLE
Can provide links of  Albert's model and sentence word model 

### ISSUE THREAD

**Issue by w279805299** [lonePatient/Bert-Multi-Label-Text-Classification#47]
Can provide links of  Albert's model and sentence word model ,Thks.

**Comment by lonePatient** [comment]
@w279805299 https://github.com/lonePatient/albert_pytorch

**Comment by w279805299** [comment]
@lonePatient Thank you ,I do not have any software to cross GFW for download the albert link. Instead of download the links of https://github.com/p208p2002/albert-zh-for-pytorch-transformers.
But I do not found  the model 'albert_vocab_path': BASE_DIR / 'pretrain/albert/albert-base/30k-clean.model' for chinese. Do you kown any open source ?
